.github/workflows/stale-issue.yml

```
@@ -22,7 +22,7 @@ jobs:
 
           # ISSUES ------------------------------------------------------------
           days-before-issue-stale: 365
-          exempt-issue-labels: epic
+          exempt-issue-labels: "epic,from:contributor"
```

```suggestion
          exempt-issue-labels: "epic"
```

For issues, we shouldn't exempt contributors.